imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -23,20 +25,25 @@ class CaseExplorer extends Component {
     this.state = {
       caseId: '',
       expandedUnits: [],
-      showOpen: true,
-      assignedToMe: false
+      filterStatus: true,
+      myInvolvement: false
     }
   }
 
-  handleStatusClicked (value) {
-    this.setState({ showOpen: value })
+  handleStatusClicked = (value) => {
+    this.setState({ filterStatus: value })
   }
 
-  handleAssignedClicked () {
-    this.setState({ assignedToMe: !this.state.assignedToMe })
+  handleMyInvolvementClicked = () => {
+    this.setState({ myInvolvement: !this.state.myInvolvement })
   }
 
-  handleExpandUnit (evt, unitTitle) {
+  handleOnItemClicked = () => {
+    const { dispatch, match } = this.props
+    dispatch(storeBreadcrumb(match.url))
+  }
+
+  handleExpandUnit = (evt, unitTitle) => {
```

Comment:
This handler, along with the state var "expandedUnits" should be moved into UnitGroupList (just to clarify my other comment)